### PR TITLE
Business Site - Laima

### PR DESCRIPTION
**Pull request by leelalaim**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
+This project of a Business Site, was made as a Week 3 project for Technigo bootcamp. 
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+The aim of this project was to create a responsive Business Site and learn how to modify the DOM using Javascript (for adding an FAQ accordion to the website, that will collapse), how to add a responsive hero image and build and style a signup form that would post the data from the form into a HTTPBin service.
 
 ## The problem
 
 Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
 
+One of the tasks hat was aimed to be achieved - adding an icon after the title in the accordion, indicating whether the section is expanded or collapsed. After trying out different ways of achieving this, on MDN Web documentations I found out, that a ::after selector can be used to achieve this.
```
Comment (HIPPIEKICK) [inline README.md:11]: Great!

File: code/index.html [hunk 2]
```diff
@@ -3,28 +3,147 @@
...
+      <img
+        class="qa-photo"
+        src="./media/photo-girl.jpg"
+        alt="girl-photographer"
+      />
     </section>
-    <!-- dont forget to link the javascript file here! -->
+
+    <!-- Signup form -->
+    <form class="form" action="https://httpbin.org/anything" method="POST">
```
Comment (HIPPIEKICK) [inline code/index.html:84]: [Here](https://www.w3schools.com/tags/tag_label.asp) you can read about why we should add `<label>`s to our inputs

File: code/index.html [hunk 3]
```diff
@@ -3,28 +3,147 @@
...
+        ></textarea>
+      </div>
+      <div class="form-item">
+        <div>
+          <label for="contact">Preferable way of contacting you:</label><br />
+          <input type="radio" id="contact" name="contact" value="email" />
+          <label for="contactChoice1">Email</label>
+
+          <input type="radio" id="contact" name="contact" value="phone" />
+          <label for="contactChoice2">Phone</label>
```
Comment (HIPPIEKICK) [inline code/index.html:127]: The `for` and `id` attributes need to correspond

File: code/index.html [hunk 4]
```diff
@@ -3,28 +3,147 @@
...
+        </div>
+      </div>
+      <div class="form-item">
+        <label
+          ><input class="checkbox" type="checkbox" required />I agree to the
+          terms
+        </label>
```
Comment (HIPPIEKICK) [inline code/index.html:134]: These should also have `for` and `id` attributes

File: code/script.js [hunk 5]
```diff
@@ -6,4 +9,7 @@ function toggle() {
 }
 
 // Selects an HTML element, and calls a function which will be executed when the element is clicked.
-document.getElementById("section1").onclick = toggle
+document.getElementById("section1").onclick = toggle;
+document.getElementById("section2").onclick = toggle;
+document.getElementById("section3").onclick = toggle;
+document.getElementById("section4").onclick = toggle;
```
Comment (HIPPIEKICK) [inline code/script.js:15]: If you decide to go with semicolons, you should have them everywhere (so add them to the function as well). Apart from that, nice and clean!

File: code/style.css [hunk 6]
```diff
@@ -1,7 +1,223 @@
...
+.question::after {    /* Plus symbol */
+  content: "\002B";
+  font-weight: bold;
+  float: right;
+  margin-left: 7px;
+}
+
+.active::after {   /* Minus symbol */
+  content: "\2212";
+}
```
Comment (HIPPIEKICK) [inline code/style.css:128]: Really nice!

**Review by HIPPIEKICK — CHANGES_REQUESTED** [review]
Good job Laima! I would just like to ask you to fix the comments I made about the `label`s. They are important for accessibility reasons, and the `for` - `id` is necessary for the connection to be made (you can see in your HTTPbin response that the checkbox and the radio button info is not there right now). Read more about this in the [general feedback document](https://www.notion.so/technigo/Week-3-Business-site-9f3ef7b1c0ee480c8a10b9c7e460c475).

**Comment by leelalaim** [comment]
Made the changes.

**Pull request closed without merging**